internal/commands/run.go

```diff
@@ -120,13 +126,18 @@ func run(cmd *cobra.Command, args []string) error {
 
 	// Container-related options
 	executorOpts = append(executorOpts, executor.WithContainerOptions(options.ContainerOptions{
-		EagerPull: !containerLazyPull,
+		EagerPull: !(lazyPull || containerLazyPull),
 		NoCleanup: debugNoCleanup,
 
 		DockerfileImageTemplate: dockerfileImageTemplate,
 		DockerfileImagePush:     dockerfileImagePush,
 	}))
 
+	// Tart-related options
+	executorOpts = append(executorOpts, executor.WithTartOptions(options.TartOptions{
+		LazyPull: lazyPull || tartLazyPull,
```

Comment:
Should it also be called `EagerPull` for consistency&

Reply:
It was called like that before https://github.com/cirruslabs/cirrus-cli/pull/517/commits/ef89082c9bdf10411b553db4b0cd41336e2f27a2, where I've changed the meaning so that each newly-initialized `TartOptions` struct would reflect the defaults.

Reply:
Should the container option be renamed into `LazyPull` then?

Reply:
See https://github.com/cirruslabs/cirrus-cli/pull/517/commits/241da4f91fda46ba9c754397076bbf2d1226c298.